src/main/java/hudson/plugins/jira/JiraChangeLogAnnotator.java

```diff
@@ -26,13 +27,36 @@
  */
 @Extension
 public class JiraChangeLogAnnotator extends ChangeLogAnnotator {
+	
     private static final Logger LOGGER = Logger.getLogger(JiraChangeLogAnnotator.class.getName());
 
+    public JiraChangeLogAnnotator()
+    {
+        super();
+        if(LOGGER.isLoggable(Level.FINE))
+        {
+        	LOGGER.info("JiraChangeLogAnnotator registered");
```

Comment:
It's not important, i only log it with fine level and use it when debuging plugin.. Ok, i change message to ""JiraChangeLogAnnotator created".